Framework/DataHandling/inc/MantidDataHandling/LoadMuonNexusV2.h

```diff
@@ -77,7 +77,8 @@ class DLLExport LoadMuonNexusV2 : public API::NexusFileLoader {
   bool m_multiPeriodsLoaded;
   // The loading strategy used
   std::unique_ptr<LoadMuonStrategy> m_loadMuonStrategy;
-}; // namespace DataHandling
-
+  // Change the time axis unit as LoadISISNexus has the wrong one
+  void applyTimeAxisUnitCorrection(API::Workspace_sptr &workspace);
```

Comment:
Since we already have ownership this should just take a bare `Workspace &` rather than a shared pointer ref